src/Functions/randomPrintableASCII.cpp

```diff
@@ -120,7 +120,30 @@ class FunctionRandomPrintableASCII : public IFunction
 
 REGISTER_FUNCTION(RandomPrintableASCII)
 {
-    factory.registerFunction<FunctionRandomPrintableASCII>();
+    FunctionDocumentation::Description description = R"(
+Generates a string with a random set of [ASCII](https://en.wikipedia.org/wiki/ASCII#Printable_characters) characters.
+All characters are printable.
+If you pass `length < 0`, the behavior of the function is undefined.
+    )";
+    FunctionDocumentation::Syntax syntax = "randomPrintableASCII(length)";
+    FunctionDocumentation::Arguments arguments = {
```

Comment:
Functions
- `randomPrintableASCII`,
- `randomString`

have an optinal 2nd argument which the markdown docs forgot to mention, see [here](https://fiddle.clickhouse.com/2bc45b28-41b6-44bb-9b9a-f330276dd91d). It has no impact on the result (at least not like the `length` argument), its only purpose is to outsmart the SQL optimizer which would otherwise apply common subexpression elimination on functions that are supposed to return random output.

See for yourself what happens here:

```sql
SELECT randomPrintableASCII(3) AS a, randomPrintableASCII(3) AS b;
```

vs. here:

```sql
SELECT randomPrintableASCII(3, 42) AS a, randomPrintableASCII(3, 43) AS b;
```